test/lucas_kanade.jl

```diff
@@ -1,151 +1,125 @@
-# We output a message after loading each package to work around a
-# ten-minute timeout limit on Travis. Travis assumes the tests have hung
-# if the interval between printing something to stdio exceeds ten minutes.
 using Images
-@info "Finished loading Images package."
 using TestImages
-@info "Finished loading TestImages package."
 using StaticArrays
-@info "Finished loading StaticArrays package."
 using OffsetArrays
-@info "Finished loading OffsetArrays package."
 using Random
-@info "Finished loading Random package."
 using CoordinateTransformations
-@info "Finished loading CoordinateTransformations package."
 
-function evaluate_error(dims, flow::Array{SVector{2, Float64}, 1},  Δ,  tol)
+function evaluate_error(dims, flow::Array{SVector{2, Float64}, 1}, Δ, tol)
     error_count = 0
     maximum_error = 0.0
     for i in eachindex(flow)
         δ = flow[i]
         if abs(δ[1] - Δ[1]) > tol || abs(δ[2] - Δ[2]) > tol
-                error_count += 1
+            error_count += 1
         end
-        error = sqrt( (δ[1] - Δ[1])^2 + (δ[2] - Δ[2])^2 )
+        error = sqrt((δ[1] - Δ[1]) ^ 2 + (δ[2] - Δ[2]) ^ 2)
```

Comment:
I wouldn't add those whitespaces there. I'd prefer to leave it as it was.

Reply:
Reverted